README.md

```diff
@@ -1,205 +1,95 @@
-
-```bash
-git clone https://github.com/meilisearch/meilisearch.git
-cd meilisearch
-cargo run --release
-```
-
-### Create an Index and Upload Some Documents
-
-Let's create an index! If you need a sample dataset, use [this movie database](https://www.notion.so/meilisearch/A-movies-dataset-to-test-Meili-1cbf7c9cfa4247249c40edfa22d7ca87#b5ae399b81834705ba5420ac70358a65). You can also find it in the `datasets/` directory.
-
-```bash
-curl -L https://docs.meilisearch.com/movies.json -o movies.json
-```
-
-Now, you're ready to index some data.
-
-```bash
-curl -i -X POST 'http://127.0.0.1:7700/indexes/movies/documents' \
-  --header 'content-type: application/json' \
-  --data-binary @movies.json
-```
-
-### Search for Documents
-
-#### In command line
-
-The search engine is now aware of your documents and can serve those via an HTTP server.
+## ✨ Features
 
-The [`jq` command-line tool](https://stedolan.github.io/jq/) can greatly help you read the server responses.
+- **Search-as-you-type:** relevant search results in less than 50 milliseconds
+- **[Typo tolerance](https://docs.meilisearch.com/learn/getting_started/customizing_relevancy.html#typo-tolerance):** Meilisearch adapts to typos and misspellings so your search results are always relevant
+- **[Filtering and faceted search](https://docs.meilisearch.com/learn/advanced/filtering_and_faceted_search.html#configuring-filters):** enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code
+- **[Sorting](https://docs.meilisearch.com/learn/advanced/sorting.html):** sort results based on price, date, or pretty much anything else your users need
+- **[Synonym support](https://docs.meilisearch.com/learn/getting_started/customizing_relevancy.html#synonyms):** configure synonyms to include more relevant content in your search results
+- **[Geosearch](https://docs.meilisearch.com/learn/advanced/geosearch.html):** filter and sort documents based on geographic data
+- **[Extensive language support](https://docs.meilisearch.com/learn/what_is_meilisearch/language.html):** Meilisearch works with all languages and offers optimized support for languages using the Latin alphabet, Chinese, Japanese, and Hebrew
+- **[Security management](https://docs.meilisearch.com/learn/security/master_api_keys.html):** control which users can access what data with API keys that allow fine-grained permissions handling
+- **[Multi-Tenancy](https://docs.meilisearch.com/learn/security/tenant_tokens.html):** personalize search results for any number of application tenants
+- **Highly Customizable:** Meilisearch's default presets work out-of-the-box but are customizable to your specific needs
+- **[RESTful API](https://docs.meilisearch.com/reference/api/overview.html):** seamlessly integrate Meilisearch in your technical stack with our plugins and SDKs
+- **Easy to install, deploy, and maintain**
```

Comment:
I feel that these catch-phrases are not consistent with each other:
- sometimes we start with `Meilisearch + VERB + ...`
> Meilisearch adapts to typos and misspellings so your search results are always relevant
- sometimes we start with `ADJ + NOUN + ...`
> relevant search results in less than 50 milliseconds
- most of the time we start with `VERB + (your) + ...`
> enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code

I'm probably a bit picky, don't hesitate to ignore this comment.

Reply:
thanks, you're absolutely right. I was trying to keep them consistent, but got sidetracked by other concerns. I will fix it ASAP

Reply:
Thank you gui!

Reply:
There, all item descriptions should start with verbs now. I also cheated and moved "Typo tolerance" a couple of items so we didn't two very similar sentences too close to each other. Let me know if that's a problem.

```suggestion
- **Search-as-you-type:** find relevant search results in less than 50 milliseconds
- **[Filtering and faceted search](https://docs.meilisearch.com/learn/advanced/filtering_and_faceted_search.html#configuring-filters):** enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code
- **[Sorting](https://docs.meilisearch.com/learn/advanced/sorting.html):** sort results based on price, date, or pretty much anything else your users need
- **[Typo tolerance](https://docs.meilisearch.com/learn/getting_started/customizing_relevancy.html#typo-tolerance):** get relevant results even when queries contain typos and misspellings
- **[Synonym support](https://docs.meilisearch.com/learn/getting_started/customizing_relevancy.html#synonyms):** configure synonyms to include more relevant content in your search results
- **[Geosearch](https://docs.meilisearch.com/learn/advanced/geosearch.html):** filter and sort documents based on geographic data
- **[Extensive language support](https://docs.meilisearch.com/learn/what_is_meilisearch/language.html):** search datasets in any language, with optimized support for Chinese, Japanese, Hebrew, and languages using the Latin alphabet
- **[Security management](https://docs.meilisearch.com/learn/security/master_api_keys.html):** control which users can access what data with API keys that allow fine-grained permissions handling
- **[Multi-Tenancy](https://docs.meilisearch.com/learn/security/tenant_tokens.html):** personalize search results for any number of application tenants
- **Highly Customizable:** customize Meilisearch to your specific needs or use our out-of-the-box and hassle-free presets
- **[RESTful API](https://docs.meilisearch.com/reference/api/overview.html):** integrate Meilisearch in your technical stack with our plugins and SDKs
- **Easy to install, deploy, and maintain**
```

Reply:
I'm pretty sure Guillaume already sorted them by "most famous" order. I don't mean your new order is not better or anything, but I don't feel really comfortable changing this order without his confirmation

So waiting for @gmourier before doing anything here 😇 
Thanks a lot @guimachiavelli for your suggestion. Depending on Guillaume's decision, I will apply the changes myself according to what you already wrote, so no worries for you!

Reply:
Can we put the "Typo tolerance" bullet point below "Search-as-you-type"? It seems important to me to mention it just after the notion of search-as-you-type.

Everything else is fine by me! 🍍